scss/_base_forms-range.scss

```diff
@@ -2,7 +2,7 @@
   $thumb-shadow: 0 0 $bar-thickness 1px rgba(0, 0, 0, 0.2);
   $thumb-size: 1rem;
   $thumb-radius: 50%;
-  $thumb-border: $input-border-thickness solid $colors--theme--border-high-contrast;
+  $thumb-border: $border-high-contrast;
```

Comment:
I would probably recommend a different name for this variable - "high contrast" traditionally means a display mode in which colors are completely inverted for users with severe vision issues.

Reply:
We are going to be revisiting colour variables when working on design tokens, so this can be addressed then.

Reply:
Removed this change.